### ISSUE TITLE
Attempt to Divide by Zero in `inflation.rs`

### ISSUE THREAD

**Issue by AurevoirXavier** [darwinia-network/darwinia#129]
```
Thread 'main-tokio-12' panicked at 'attempt to divide by zero', /Users/xavier/Documents/program/darwinia/srml/staking/src/inflation.rs:26
```

**Comment by AurevoirXavier** [comment]
@HackFisher 

```rust
// line 26
let maximum = maximum - maximum * 99_u128.pow(year.integer_sqrt()) / 100_u128.pow(year.integer_sqrt());
```

**Comment by AurevoirXavier** [comment]
Is this relate to #103?

Both of them happens at block-height 1000.

When I delete line 26, #103 disappeared.

**Comment by HackFisher** [comment]
> @HackFisher, [line 26](https://github.com/darwinia-network/darwinia/blob/4f3b6f309459a77fce2cabc79d69a97af76c46d7/srml/staking/src/inflation.rs#L26):
> 
> ```rust
> let maximum = maximum - maximum * 99_u128.pow(year.integer_sqrt()) / 100_u128.pow(year.integer_sqrt());
> ```

I have no idea, this line related to inflation supply calculation(refer genepaper).

```100_u128.pow(year.integer_sqrt())``` It looks this value can not be zero, some tests might help

**Comment by AurevoirXavier** [comment]
Fixed: #133.
